app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,12 @@
+class CommentsController < ApplicationController
+ def create
+    @post = Post.find(params[:post_id])
```

Comment:
alignment issue

Reply:
This is Rubocop auto corrected file

Reply:
need 2 space before def I think. Can you run rubocop auto correct once more?